src/UserSpaceInstrumentation/TestLib.cpp

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "TestLib.h"
+
+#include <cstdio>
+#include <filesystem>
+#include <fstream>
+#include <iostream>
+
+namespace orbit_user_space_instrumentation {
+
+namespace {
+
+std::filesystem::path GetTmpFilePath() {
+  static std::filesystem::path p = std::tmpnam(nullptr);
+  return p;
+}
+
+}  // namespace
+
+void InitTestLib() {
+  auto p = GetTmpFilePath();
+  std::cout << "Init Lib. Tmp file is: " << p << std::endl;
+}
+
+void UseTestLib(const std::string& s) {
```

Comment:
Not that it matters, but I guess you can use `string_view` in this case.

Reply:
done